app/routers/dayview.py

```diff
@@ -26,6 +26,41 @@ class DivAttributes:
     CLASS_SIZES = ("title-size-tiny", "title-size-xsmall", "title-size-small")
     LENGTH_SIZE_STEP = (30, 45, 90)
 
+    def _minutes_position(self, minutes: int) -> dict[str, int]:
+        min_minutes = self.MIN_MINUTES
+        max_minutes = self.MAX_MINUTES
+        for i in range(self.GRID_BAR_QUARTER, self.FULL_GRID_BAR + 1):
+            if min_minutes < minutes <= max_minutes:
+                minute_deviation = minutes - (i - 1) * self.MAX_MINUTES
+                return {"min_position": i, "min_deviation": minute_deviation}
+            min_minutes = max_minutes
+            max_minutes += self.MAX_MINUTES
+
+    def _get_position(self, time: datetime) -> int:
+        grid_hour_position = time.hour * self.FULL_GRID_BAR
+        grid_minutes_modifier = self._minutes_position(time.minute)
+        if grid_minutes_modifier is None:
+            grid_minutes_modifier = 0
+        else:
+            grid_minutes_modifier = grid_minutes_modifier["min_position"]
+        return grid_hour_position + grid_minutes_modifier + self.BASE_GRID_BAR
+
+
+class CurrentTimeAttributes(DivAttributes):
+    def __init__(self, date: datetime) -> None:
+        current = datetime.now()
+        self.dayview_date = date.date()
+        self.is_viewed = self._date_is_today()
+        self.grid_position = self._get_position(current) - 1
+        self.sub_grid_position = self._minutes_position(current.minute)
+        self.sub_grid_position = self.sub_grid_position["min_deviation"]
+
+    def _date_is_today(self) -> bool:
+        today = datetime.now().date()
```

Comment:
```suggestion
        from datetime import datetime
        datetime.today()
```

Reply:
I changed it! although it is still needed to be called as datetime.today().date()